tests/test_stream.py

```diff
@@ -0,0 +1,41 @@
+from lomond.stream import WebsocketStream, FrameParser
+from lomond.errors import ProtocolError
+import pytest
+
+
+@pytest.fixture
+def stream():
+    return WebsocketStream()
+
+
+def test_constructor(stream):
+    assert isinstance(stream, WebsocketStream)
+
+
+def test_feed(stream):
+    data = (
+        b'Connection:Keep-Alive\r\nUser-Agent:Test\r\n\r\n'
+        b'\x81\x81\x00\x00\x00\x00A'
+    )
+
+    feed = list(stream.feed(data))
+
+    assert len(feed) == 2
```

Comment:
I agree to the second part (elaborating on the items). However, I have explainend very verbosely what the binary codes are in test_frame_parser.py, so I think a fair compromise would be that I would do a brief explanation + a reference to search for in-depth explanation in test_frame_parser.py